test/package/packageTest.nut.ts

```diff
@@ -369,13 +367,6 @@ describe('Integration tests for @salesforce/packaging library', function () {
       ]);
     });
 
-    it('gets an error trying to uninstall again (and waiting for the result)', () => {
```

Comment:
this command doesn't have a `--wait` flag, and so it was always getting `exitCode:1`